go/consensus/tendermint/apps/registry/genesis.go

```diff
@@ -29,18 +29,6 @@ func (app *registryApplication) InitChain(ctx *abci.Context, request types.Reque
 	state := registryState.NewMutableState(ctx.State())
 	state.SetConsensusParameters(&st.Parameters)
 
-	if st.Parameters.KeyManagerOperator.IsValid() {
```

Comment:
I'm still totally against this.

Reply:
Added a separate consensus parameter to allow explicitly disabling key manager runtime registrations outside genesis. Other than that #2642 also introduces a way to specify a staking threshold for key manager runtime registrations.